src/ResourceManager/KeyVault/Commands.KeyVault.Test/Scripts/ControlPlane/KeyVaultManagementTests.ps1

```diff
@@ -37,6 +37,8 @@ Param($rgName, $location, $tagName, $tagValue)
     Assert-AreEqual "Standard" $actual.Sku	
     Assert-AreEqual $false $actual.EnabledForDeployment
 
+    if ($global:noADCmdLetMode) {return;}
```

Comment:
I really don't like this.  This just skips the test without letting us know that's what happened.  A better solution would be to store the needed subscriptionId in the mocks, which is how most tests solve similar issues.